docker-compose.yml

```diff
@@ -63,6 +63,7 @@ services:
       - CLAMAV_HOST=${CLAMAV_HOST}
       - ENABLE_FILE_VIRUS_SCAN=${ENABLE_FILE_VIRUS_SCAN}
       - GCNOTIFY_SECRET_API_KEY=${GCNOTIFY_SECRET_API_KEY}
```

Comment:
One thing I forgot about, is that we also need to add these env vars to our deployments scripts, otherwise they will be undefined when we deploy to openshift (the docker-compose file is only used for local dev). Can chat about it tomorrow.